### PR TITLE
Fix small misspellings

### PR DESCRIPTION
**Pull request by MisterDA**
Fix small misspellings.
They were found by https://github.com/client9/misspell.

### CODE REVIEW

File: doc/pages/External_solvers.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ When you request to install a (set of) package(s), in general you do not expect
 ### Specifying preferences for opam
 
 `opam` allows to specify your criteria on the command line, using the `--criteria` option, that will apply only to the current command.
-For example if you are a very conservative user, you might try issueing the following command:
+For example if you are a very conservative user, you might try issuing the following command:
```
Comment (dra27) [inline doc/pages/External_solvers.md]:
```suggestion
For example, if you are a very conservative user, you might try issuing the following command:
```

File: doc/pages/Specifying_Solver_Preferences.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ When you request to install a (set of) package(s), in general you do not expect
 ### Specifying preferences for opam
 
 Recent versions of `opam` allow to specify your criteria on the command line, using the `--criteria` option, that will apply only to the current command.
-For example if you are a very conservative user, you might try issueing the following command:
+For example if you are a very conservative user, you might try issuing the following command:
```
Comment (dra27) [inline doc/pages/Specifying_Solver_Preferences.md]:
```suggestion
For example, if you are a very conservative user, you might try issuing the following command:
```

File: tests/README.unittest [hunk 3]
```diff
@@ -9,7 +9,7 @@
 
       ./init-repo.sh -i
 
-* load a inital scenario (this command can be invoked multiple times): 
+* load a initial scenario (this command can be invoked multiple times):
```
Comment (dra27) [inline tests/README.unittest]:
```suggestion
* load an initial scenario (this command can be invoked multiple times): 
```

**Comment by MisterDA** [comment]
Hop! I added your suggestions.

**Comment by rjbou** [comment]
Thanks!

**Pull request merged**